user-creation/src/test/java/no/unit/nva/useraccessservice/usercreation/PersonInformationTest.java

```diff
@@ -0,0 +1,22 @@
+package no.unit.nva.useraccessservice.usercreation;
+
+import static no.unit.nva.testutils.RandomDataGenerator.randomString;
+import static no.unit.nva.testutils.RandomDataGenerator.randomUri;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsEqual.equalTo;
+import java.net.URI;
+import no.unit.nva.useraccessservice.usercreation.cristin.person.CristinPersonResponse;
+import org.junit.jupiter.api.Test;
+
+class PersonInformationTest {
+
+    @Test
+    void shouldReturnPersonRegistryIdWhenCristinResponseExists() {
+        var personInformation = new PersonInformationImpl(randomString(), randomString());
```

Comment:
The method getPersonRegistryId is used outside the module where this method is defined. This is a